doc/Language/math.pod6

```diff
@@ -1,12 +1,12 @@
 =begin pod :tag<perl6>
 
-=TITLE Doing math with Perl 6
+=TITLE Doing math with Perl 6
 
 =SUBTITLE Different mathematical paradigms and how they are implemented in this language
 
 =head1 Sets
 
-Perl 6 includes the L<Set> data type, as well as support for
+Perl 6 includes the L<Set> data type, as well as support for
```

Comment:
If this is a whitespace, please fix it. Perl is always separated by a no-break space from 6 (same as 5), so that it's no separated by line breaks and so on. Can you please fix that?